doc/HACKING/ReleaseSeriesLifecycle.md

```diff
@@ -48,3 +50,49 @@ deprecations of individual versions.
    number from their approved versions list.
 
 7. Update the CoreTorReleases wiki page.
+
+8. Update the git/hooks
+
+9. Finally, make sure this document is up to date with our latest
+   process.
+
+10. Open a ticket (if there is not one already) for authorities to
+    start rejecting relays that are running that release series.
+    This ticket should be targeted for at least a month or two
+    after the series is officially EOL, unless there is an important
+    reason to un-list relays early.
+
+11. Open a ticket (if appropriate) for updates to the set of
+    required and recommended subprotocol versions.
```

Comment:
Should we mark this as "LTS end of life only" ?

Reply:
Let's also link to:
https://gitweb.torproject.org/torspec.git/tree/proposals/303-protover-removal-policy.txt